docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+# Deploying a production network
```

Comment:
Is it worth including a 'expected level of understanding' at the start?

Reply:
The next sentence says a 'funtioning Fabric network' but doesn't cover channels..  so is that accurate?  Can't really do a lot without a channel :-)

Reply:
Agreed --- I've changed this from "functioning" to production --- and clarified that we're talking about components here.